src/virtual-machine.js

```diff
@@ -431,7 +442,7 @@ class VirtualMachine extends EventEmitter {
 
         const runtime = this.runtime;
         const deserializePromise = function () {
-            const projectVersion = projectJSON.projectVersion;
+            let projectVersion = projectJSON.projectVersion;
```

Comment:
This change is unnecessary since projectVersion does not seem to be re-defined below.

Reply:
Nice catch. This is from any early change when I had the code using a different path to detect and transform to sb2.